pkgs/top-level/ocaml-packages.nix

```diff
@@ -230,6 +230,8 @@ let
 
     csv-lwt = callPackage ../development/ocaml-modules/csv/lwt.nix { };
 
+    csvtool = callPackage ../development/ocaml-modules/csv/csvtool.nix { };
```

Comment:
If the behavior of the tool does not depend on the version of OCaml used for compiling it, it may be better to move this attribute up into `pkgs/top-level/all-packages.nix`.

Reply:
Ok, created the entry in all-packages.

Reply:
What about removing `csvtool` from `ocamlPackages`?

Reply:
It depends on `buildDunePackage`, I kept it here so that `callPackage` find all the scope.

Reply:
I’ve just pushed what I meant. Feel free to drop my commit if you don’t like it.

Reply:
ok thanks.